src/GlobalPayments.Api/Terminals/Builders/TerminalManageBuilder.cs

```diff
@@ -32,10 +53,25 @@ public class TerminalManageBuilder : TerminalBuilder<TerminalManageBuilder> {
             Gratuity = amount;
             return this;
         }
+
+        /// <summary>
+        /// Sets the authorization code for the transaction.
+        /// </summary>
+        /// <param name="value">Authorization Code</param>
+        /// <returns></returns>
+        public TerminalManageBuilder WithAuthCode(string value) {
+            if (PaymentMethod == null || !(PaymentMethod is TransactionReference))

```

Comment:
This is not concise code. And all code blocks in a IF statement should be wrapped in braces even if it's only one line.